ssas/systems.go

```diff
@@ -444,36 +499,47 @@ func RegisterSystem(clientName string, groupID string, scope string, publicKeyPE
 			return creds, errors.New("internal system error")
 		}
 	}
+	if isV2 {
+		err = system.SaveClientToken(uuid.NewRandom().String(), "Initial Token")
+		if err != nil {
+			regEvent.Help = fmt.Sprintf("could not save client token for clientID %s, groupID %s: %s", clientID, groupID, err.Error())
+			OperationFailed(regEvent)
+			return creds, errors.New("internal system error")
```

Comment:
Should we roll back the transaction too?

Reply:
yep, fixed.